packages/plugin-vue/README.md

```diff
@@ -62,6 +62,9 @@ export default {
       template: {
         compilerOptions: {
           // ...
+        },
+        transformAssetUrls: {
+          'custom-img': ['src']
```

Comment:
This is a "legacy option format" per the comments in the Vue codebase.  Please put it inside a `tags` field.
And seems a custom configuration will override the default `tags` option, so I think it would make more sense to include all the default ones in the example, too: https://github.com/vuejs/core/blob/5898629d723e82b68e9b17b91bf8b1a8390a3912/packages/compiler-sfc/src/templateTransformAssetUrl.ts#L40-L46

Reply:
Good call. I'll probably put all those and remove the `custom-img` thing

Reply:
Done!